assignments/pr_review_tutorial/chai/README.md

```diff
@@ -1,11 +1,10 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+The simple_functions package includes several common and helpful mathematic series and functions. Here's an example of using a factorial function to find the value of 10!:
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
+    ##3628800
```

Comment:
No need to comment this, the code snippets are expected to be copy pasteble to a python interpreter and the output should be exactly what comes out from that interpreter
```suggestion
    3628800
```